api/fbs/v1/rpc.fbs

```diff
@@ -16,10 +16,12 @@
 // under the License.
 
 include "query.fbs";
+include "write.fbs";
 
 namespace banyandb.v1;
 
-// Query is the RPC service exposed
-rpc_service Query {
-    QueryTraces(EntityCriteria):TracesResponse;
-}
\ No newline at end of file
+// Trace is the RPC service exposed
+rpc_service Trace {
+    QueryTraces(EntityCriteria): TracesResponse;
```

Comment:
Since the service already has the name `Trace`, do you still need to duplicate `Traces` in the method?

Reply:
Make sense, should be query and write.

Reply:
+1